Makefile

```diff
@@ -2,17 +2,17 @@
 .PHONY: build clean test
 
 build:
-	jbuilder build @install --dev
+	dune build @install
 
 test:
-	jbuilder build --dev test/client_test.exe
-	jbuilder build --dev test/server_test.exe
+	dune build test/client_test.exe
```

Comment:
Does a `dune runtest` work instead of the 2 commands?

Reply:
we build some executables here, so `runtest` is not what we want

Reply:
`runtest` builds executables before running them. I think running the executables is also warranted when running `make test`

Reply:
A good way to do this:
```
PROFILE=release

all:
  dune build @all --profile=$(PROFILE)
```

This can be overridden from the command line: `make PROFILE=dev all`